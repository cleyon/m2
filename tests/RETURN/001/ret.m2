@newcmd foo
Start
@for i 1 5
In loop, i=@i@
@if i > 2
@return
@fi
@next i
End, not reached
@endcmd
Foo
@foo
All done!
