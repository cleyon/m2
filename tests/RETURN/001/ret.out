Foo
Start
In loop, i=1
In loop, i=2
In loop, i=3
All done!
